src/rpc/rawtransaction.cpp

```diff
@@ -366,6 +367,100 @@ static RPCHelpMan getrawtransactionmulti() {
     };
 }
 
+static RPCHelpMan getislocks()
+{
+    return RPCHelpMan{"getislocks",
+        "\nReturns the raw InstantSend lock data for each txids. Returns Null if there is no known IS yet.",
+        {
+            {"txids", RPCArg::Type::ARR, RPCArg::Optional::NO, "The transaction ids (no more than 100)",
+                {
+                    {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED, "A transaction hash"},
+                },
+            },
+            {"verbose", RPCArg::Type::NUM, RPCArg::Default{1}, "0 for hex-encoded data, 1 for a json object"},
```

Comment:
nit: could be `bool`

Reply:
see see RPC getblock.

Reply:
Same suggestion like for `getbestchainlock`